test/models/pregnancy_test.rb

```diff
@@ -40,12 +40,40 @@ class PregnancyTest < ActiveSupport::TestCase
 
     it 'should respond to history methods' do
       assert @pregnancy.respond_to? :history_tracks
-      assert @pregnancy.history_tracks.count > 0 
+      assert @pregnancy.history_tracks.count > 0
     end
 
-    it 'should have accessible userstamp methods' do 
+    it 'should have accessible userstamp methods' do
       assert @pregnancy.respond_to? :created_by
       assert @pregnancy.created_by
     end
-  end  
+  end
+
+  describe 'status method' do
+    it 'should default to "No Contact Made"' do
+      assert_equal 'No Contact Made', @pregnancy.status
+    end
+    it 'should update to "Needs Appointment" once patient has been reached' do
+      create :call, pregnancy: @pregnancy
```

Comment:
Two suggestions here:

* Add a Left Message call and confirm that the status stays at No Contact Made, so we can be sure that the existence of a call in general isn't updating it to Needs Appointment
* Clarify that the call here on L57 is a 'Reached patient' call; e.g. `create :call, pregnancy: @pregnancy, status: 'Reached patient'` so we can tell just by looking at the test what's going on, rather than popping into the factory